ament_cmake_core/cmake/core/python.cmake

```diff
@@ -12,12 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set(PYTHON_VERSION "" CACHE STRING
-  "Specify specific Python version to use ('major.minor' or 'major')")
-# if not specified otherwise use Python 3
-if(NOT PYTHON_VERSION)
-  set(PYTHON_VERSION "3")
+# ament_cmake needs a Python 3 interpreter
+# If a specific Python version is required then find it before finding ament_cmake
+# Example:
+#   find_package(Python3 3.8 REQUIRED)
+#   find_package(ament_cmake REQUIRED)
```

Comment:
It does matter since the code here only finds the interpreter if the target doesn't already exist. Any ideas where it should be documented? The current code documents the equivalent feature only in the CMake cache.

Reply:
mmm, I'm not sure, ament_cmake doesn't have lot's of docs and it's particularly hard to document things that happen automatically when you find_package a package.

It would be good to keep an status message here about trying to find-package Python3::Interpreter and the path (and version) of the python executable found, so at least people can find that in the logs.
Maybe `FindPython3` is already logging that, in that case we don't need to do anything here.

Reply:
>  Maybe FindPython3 is already logging that, in that case we don't need to do anything here.

Yup, `FindPython3` is already logging that. Here's what it looks like:

```
-- Found Python3: /usr/bin/python3.8 (found version "3.8.10") found components: Interpreter 
```